packages/internal-lib-build/configs/jest/jest.config.js

```diff
@@ -28,7 +28,6 @@ module.exports = {
         '/node_modules/',
         '/vendor/'
     ],
-    testRegex: '\\.?(test|spec)\\.jsx?$',
```

Comment:
Deleting this, and letting the default value takes care of covering both javascript and typescript file extensions.

See: https://jestjs.io/docs/26.x/configuration#testregex-string--arraystring